client/my-sites/plugins/marketplace/util.ts

```diff
@@ -0,0 +1,17 @@
+//Navigation functions
+export function navigateToProductHomePage(
+	page: ( url: string ) => void,
+	selectedSiteSlug = '',
+	pluginSlug = 'wordpress-seo'
+): void {
+	page(
+		`/marketplace/product/details/${ pluginSlug }/${ selectedSiteSlug }?flags=marketplace-yoast`
```

Comment:
It seems like we're not handling the case when `selectedSiteSlug` is an empty string. In that instance, the trailing slash shouldn't be added after the `pluginSlug`.

Reply:
Actually I thought of relying on typescript's type checking here to not accept empty strings. Or do we have to handle those cases as well @tyxla ?

Reply:
Um, how would TS prevent from empty strings, given that the default value of the argument is an empty string?

Reply:
I think we might be looking at different versions, a github glitch possibly, but I removed the default values earlier.

https://github.com/Automattic/wp-calypso/blob/773efc31b1d6a8b96dd6ee73f7dc898343380b4b/client/my-sites/plugins/marketplace/util.ts#L10-L18

Reply:
Gotcha. Yep, that should do the job. AFAIK we can't have an empty string for a selected site slug unless something went terribly wrong. It'll either be a real slug, or a `null`.